public/pages/AnomalyCharts/index.scss

```diff
@@ -1,18 +1,3 @@
-/*
```

Comment:
Add new copyright header and check other files?

Reply:
Got it will check other files, didn't realize some files didn't have the other headers

Reply:
@ylwu-amzn @ohltyler Should I add a copyright header to files like `package.json` and `.prettierrc` also? A few more config files never had at any point a copyright header

Reply:
If possible, sure. I remember there was some issue adding comments to some of those files, but if it compiles and works, go ahead.

Reply:
You are right, can't add comments to json files and some others